### ISSUE TITLE
Update documentation for spibb

### ISSUE THREAD

**Issue by yzgyyang** [evadot/fbsd_gpio_py#1]
Hi! I used this library for my project and wrote [this article](https://github.com/yzgyyang/guide/blob/master/BeagleBone_FreeBSD_SPI_Setup.md). I implemented the spi bit banging functions myself without realizing there exists an abstraction in this library! I already mentioned this in my post, and am wondering if you could update documentations for spibb when you are free so that more people can see and use it?

**Comment by evadot** [comment]
Yes I have branch somewhere when I started some real documentation that I need to polish and publish somewhere.
